resources/views/admin.blade.php

```diff
@@ -169,7 +169,7 @@ function deleteEmployee(id){
                         $('#email').val(data.email)
                         $('#phone').val(data.phone)
                         $('#address').val(data.address)
-                        $(`#option-id-${data.company.id}`).attr('selected', true);
+                        $(`#select option[value="${data.company.id}"]`).attr('selected', true);
```

Comment:
refactor this method. use the element `id` and setup the value for select element

Reply:
as requested used element id to setup up the value please review again @Raza9798